### PR TITLE
KFAX-23 : Expose parameters which required to determine file upload file size

### PR DESCRIPTION
**Pull request by wdhammika**
Previously, Support did not know what the monster UI should be updated to,  Currently, we have 3 different answers, comm.land states 15mb, the documentation states 3mb, and eFax in Kazoo report 20 MB. We are trying to track down the correct size

Two configuration parameters have been identified which determine file size.
```
crossbar => max_upload_size
fax => smtp_max_msg_size
```
Therefore `v2/accounts/{ACCOUNT_ID}/configs` has been configured to return parameters that UI needed to determine the file size and others. So UI can access those values through API.

### CODE REVIEW

File: src/modules/cb_configs.erl [hunk 1]
```diff
@@ -43,16 +43,24 @@ init() ->
     _ = crossbar_bindings:bind(<<"*.execute.patch.configs">>, ?MODULE, patch),
     _ = crossbar_bindings:bind(<<"*.execute.delete.configs">>, ?MODULE, delete).
 
+-spec allowed_methods() -> http_methods().
+allowed_methods() ->
+    [?HTTP_GET, ?HTTP_POST, ?HTTP_PUT, ?HTTP_PATCH, ?HTTP_DELETE].
```
Comment (jamesaimonetti) [inline src/modules/cb_configs.erl]: I don't think this API should allow anything but GET, right?

**Review by jamesaimonetti — CHANGES_REQUESTED** [review]
Want to see properly test too, to verify response data

File: src/modules/cb_configs.erl (hunk 1 above)
Comment (wdhammika) [reply src/modules/cb_configs.erl]: Yes, GET is the main intention of this, but I have enabled other options to save/update parameters.

Comment (jamesaimonetti) [reply src/modules/cb_configs.erl]: These configs are global in nature and no account should be able to modify them. Please remove all but GET

Comment (wdhammika) [reply src/modules/cb_configs.erl]: Sure! I will remove them.

File: priv/api/swagger.json [hunk 2]
```diff
@@ -397,6 +397,22 @@
             },
             "type": "object"
         },
+        "account_config.default": {
+            "description": "Schema for default account_config",
+            "properties": {
+                "max_upload_size": {
+                    "default": 20,
+                    "description": "maximum upload file size",
```
Comment (harenson) [inline priv/api/swagger.json:405]: What measurement unit is being used? b, Kib, Mib, Gib, etc? Please, include it within the description. Something like: `"maximum (in {MEASURE}) upload file size"`.

Comment (wdhammika) [reply priv/api/swagger.json:405]: Fixed.

File: doc/ref/configs.md [hunk 3]
```diff
@@ -6,6 +6,16 @@
 
 
 
+## Fetch
```
Comment (jamesaimonetti) [inline doc/ref/configs.md:9]: Bring this doc change down a dir to the `doc/configs.md` and add a sample response JSON payload

Comment (wdhammika) [reply doc/ref/configs.md:9]: This is completed.

File: src/modules/cb_configs.erl (hunk 1 above)
Comment (wdhammika) [reply src/modules/cb_configs.erl]: Added Properly test cases.
https://github.com/2600hz/kazoo-properly/pull/147

**Pull request merged**